### ISSUE TITLE
'RoutingHTTPServer/RoutingConncection.h' file not found

### ISSUE THREAD

**Issue by kgneil** [facebook/WebDriverAgent#541]
Error in WebDriverAgentLib/Routing/FBWebServer.m

Line '#import <RoutingHTTPServer/RoutingConnection.h>'
Is causing an error because the file is not found. I did a pull this morning because last night I was getting the open url error.
Xcode 8.3

**Comment by marekcirkos** [comment]
Did you run bootstrap.sh script?

**Comment by kgneil** [comment]
whoops I forgot to redo all of that. Thank you

**Comment by sarozNep** [comment]
@marekcirkos  i have run the bootstrap.sh but have the build error too ... what to do

**Comment by jerzynowakowski** [comment]
In folder:
`/usr/local/lib/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent`
execute command:
`./Scripts/bootstrap.sh -d`

**Comment by fisagirl** [comment]
How to run
./Scripts/bootstrap.sh -d

**Comment by fisagirl** [comment]
Last login: Thu Jan 11 14:21:58 on ttys000
fanniedeMacBook-Pro:~ fannie$ ./Scripts/bootstrap.sh
-bash: ./Scripts/bootstrap.sh: No such file or directory
fanniedeMacBook-Pro:~ fannie$ #!/bin/bash
fanniedeMacBook-Pro:~ fannie$ #
fanniedeMacBook-Pro:~ fannie$ # Copyright (c) 2015-present, Facebook, Inc.
fanniedeMacBook-Pro:~ fannie$ # All rights reserved.
fanniedeMacBook-Pro:~ fannie$ #
fanniedeMacBook-Pro:~ fannie$ # This source code is licensed under the BSD-style license found in the
fanniedeMacBook-Pro:~ fannie$ # LICENSE file in the root directory of this source tree. An additional grant
fanniedeMacBook-Pro:~ fannie$ # of patent rights can be found in the PATENTS file in the same directory.
fanniedeMacBook-Pro:~ fannie$ #
fanniedeMacBook-Pro:~ fannie$ 
fanniedeMacBook-Pro:~ fannie$ set -e
fanniedeMacBook-Pro:~ fannie$ 
fanniedeMacBook-Pro:~ fannie$ export PATH=$PATH:/usr/local/bin
fanniedeMacBook-Pro:~ fannie$ BOLD="\033[1m"
fanniedeMacBook-Pro:~ fannie$ 
fanniedeMacBook-Pro:~ fannie$ if [[ ! -f Scripts/bootstrap.sh ]]; then
>   echo "Run this script from the root of repository"
>   exit 1
> fi
Run this script from the root of repository
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[进程已完成]
I runed bootstrap successfully,but still have the same problem 
pleaseeeee, help

**Comment by SamyQAZ** [comment]
@fisagirl same problem here updated Appium CLI to 1.3.0,restart machine.
So I just copied WDA to applications directory from usr directory and now it works however Xcode project under applications still fails after complete configuration mentioned in the documentation. :worried:

**Comment by fisagirl** [comment]
I find one solution like change the webpack.config.js from 
...
loaders: [
      { test: /\.js?$/, loaders: ['babel-loader'], exclude: /node_modules/ },
      { test: /\.css?$/, loader: 'style-loader!css-loader' },
    ]
...
to
...
loaders: [
      { test: /\.js?$/, loaders: ['babel-loader'] },
      { test: /\.css?$/, loader: 'style-loader!css-loader' },
    ]
...
but I don't know how ?

**Comment by hliulka15** [comment]
I have the same issue. Successfully ran boostrap but nothing changed. Also the xcode project for WebDriverAgent on my machine has several errors from what appears to be the WebDriverAgent.bundle. I have tried the command `mkdir -p /usr/local/lib/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/Resources/WebDriverAgent.bundle` but nothing happens. I've also assigned a Provisioning Profile for WebDriverAgentRunner. 

<img width="275" alt="screen shot 2018-03-01 at 12 42 31 pm" src="https://user-images.githubusercontent.com/34134820/36863052-3b63bb96-1d4e-11e8-940f-7adb19f53c5e.png">
